op-bindings/predeploys/legacy_addresses.go

```diff
@@ -0,0 +1,11 @@
+package predeploys
+
+import "github.com/ethereum/go-ethereum/common"
+
+const (
+	LegacyERC20ETH = "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000"
```

Comment:
Just a comment I'm sure this is just following convention, but seems unnecessary and also slightly confusing to make this one public

Reply:
It needs to be public so that other packges can reference it. Otherwise we can't reference it from the indexer package

Reply:
Why would we need to reference this though instead of the geth Address object thing?  Doesn't the geth Address object have access to this string?

Reply:
ahh i see what you're saying, yea the convention in the other file follows this format. And I think the constant probably gets inlined on builds whenever it's used making it preferrable for arguments that take the string form